src/pytorch_lightning/utilities/migration/migrations.py

```diff
@@ -0,0 +1,69 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
shall this rather be:
`src/pytorch_lightning/utilities/migrations/migration.py`

Reply:
I named it plural because this file is mean to hold all the migration functions that apply. No strong opinion, happy to change